src/servers/shared_memory_manager.cc

```diff
@@ -67,39 +61,41 @@ MapSharedMemory(
   *mapped_addr = mmap(NULL, byte_size, PROT_WRITE, MAP_SHARED, shm_fd, offset);
   if (*mapped_addr == MAP_FAILED) {
     LOG_VERBOSE(1) << "mmap failed, errno: " << errno;
-    return Status(
-        RequestStatusCode::INTERNAL, "Unable to process address space");
+    return TRTSERVER_ErrorNew(
+        TRTSERVER_ERROR_INTERNAL,
+        std::string("Unable to process address space").c_str());
   }
 
-  return Status::Success;
+  return nullptr;
 }
 
-Status
+TRTSERVER_Error*
 CloseSharedMemoryRegion(int shm_fd)
 {
   int tmp = close(shm_fd);
   if (tmp == -1) {
-    return Status(
-        RequestStatusCode::INTERNAL, "Unable to close shared memory region");
+    return TRTSERVER_ErrorNew(
+        TRTSERVER_ERROR_INTERNAL,
+        std::string("Unable to close shared memory region").c_str());
   }
-
-  return Status::Success;
+  return nullptr;
 }
 
-Status
+TRTSERVER_Error*
 UnmapSharedMemory(void* mapped_addr, size_t byte_size)
 {
   int tmp_fd = munmap(mapped_addr, byte_size);
   if (tmp_fd == -1) {
```

Comment:
Change the `tmp`s to more descriptive name.. I don't think unmap call will return a fd...

Reply:
Done

Reply:
You're right, the returned value is an indicator is success (0 is success, -1 for failure)